llvm/lib/Target/PowerPC/PPCAsmPrinter.cpp

```diff
@@ -1292,8 +1291,9 @@ void PPCAsmPrinter::emitInstruction(const MachineInstr *MI) {
 
     unsigned Op = MI->getOpcode();
 
-    // Change the opcode to load address for tocdata
-    TmpInst.setOpcode(Op == PPC::ADDItocL8 ? PPC::ADDI8 : PPC::LA);
+    // Change the opcode to load address for toc data.
+    unsigned NewOp64 = IsAIX ? PPC::LA8 : PPC::ADDI8;
```

Comment:
Minor nit: Roll selecting the new opcode into a single statement rather then selecting what the opcode would be for 64-bit, then deciding too use LA or the 64-bit opcode in 2 different statements?